hello.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <div>Hello Microverse!</div>
```

Comment:
- [ ] Good job adding `Hello Microverse` as a text 😄, but if you check the project requirements it should be added as inside of an `<h1>` tag, kindly, replace the semantic tags 🌊 .
- [ ] In the requirements also is requested a header 💯 , It would be a good idea to use a `<header>` tag to do your code more readable, please would you add the `<h1>` tag inside of `<header>` tag? 🔢.